src/kinetics/KineticsFactory.cpp

```diff
@@ -168,17 +171,44 @@ void addReactions(Kinetics& kin, const AnyMap& phaseNode, const AnyMap& rootNode
             AnyMap reactions = AnyMap::fromYamlFile(fileName,
                 rootNode.getString("__file__", ""));
             for (const auto& R : reactions[node].asVector<AnyMap>()) {
-                kin.addReaction(newReaction(R, kin));
+                try{
+                    kin.addReaction(newReaction(R, kin));
+                }
+                catch(CanteraError& err){
+                    format_to(err_Plog_reactions, "{}", err.what());
```

Comment:
It seems like this will catch any `CanteraError`s that are generated by trying to add the reaction, not just errors related to Plog reactions. In that case, the name of the variable seems to be a bit of a misnomer. If you only want to capture the Plog reactions here, then I think you need some of the same string processing as you've got in the next section.

To be clear though, I think it might be a good thing to catch all the errors and report them all at once back to the user, not just the errors related to adding PLOG reactions. I'd like to try it out and see how that works from the user side though, first. I also realize that might be expanding the scope of this change quite a bit, so this is more of a "thinking-out-loud" paragraph 😄